aepp/configs.py

```diff
@@ -187,6 +198,7 @@ def configure(
     config_object["private_key"] = private_key
     config_object["auth_code"] = auth_code
     config_object["sandbox"] = sandbox
+    config_object["sslVerification"] = ssl_verification
```

Comment:
Can you also add that in the `connectInstance` class. 
It is used to save the configuration loaded into an instance in order to re-use it later on with all the sub module. 

I would imagine we need to add it in 3 places: 
*  as an attrbute (line 319)
* in the dictionary definition saved internally `self.__configObject__` (starting line 320)
* you may want to create a setter function to change this value, the way I did for sandbox.(up to you)